tests/test_imageSimulator.py

```diff
@@ -0,0 +1,46 @@
+"""
+Unit tests for the ImageSimulator module.
+"""
+import os
+import random
+import string
+import unittest
+import gzip
+import desc.imsim
+
+class ImageSimulatorTestCase(unittest.TestCase):
```

Comment:
Since these tests could in principle be more generic for the ImageSimulator module (testing things like the multiprocessing) I think it would be good to add a comment in setup and teardown that gzip functionality is specifically being tested here. Alternatively, this could be made a test specifically for compression.